.config/babel.mjs

```diff
@@ -0,0 +1,13 @@
+export default {
+  'plugins': ['@babel/plugin-syntax-top-level-await'],
+  'presets': [
+    [
+      '@babel/preset-env',
+      {
+        'targets': {
+          'esmodules': true
```

Comment:
```suggestion
  plugins: ['@babel/plugin-syntax-top-level-await'],
  presets: [
    [
      '@babel/preset-env',
      {
        targets: {
          esmodules: true
```

Reply:
Strange that this wasn't picked up by Prettier since we have defined `quoteProps: 'as-needed'` in the ESLint config file.

Reply:
Oh, I just remembered by this happens! Because we need to tell ESLint not to ignore dotfiles.